MedtrumKit/Packets/Enums/AlarmSettings.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  AlarmSettings.swift
+//  MedtrumKit
+//
+//  Created by Bastiaan Verhaar on 02/03/2025.
+//
+
+enum AlarmSettings: UInt8 {
+    case LightVibrateAndBeep = 0
```

Comment:
Older models had light and vibrate, everything that is now on market doesn't have that anymore, something to consider, fine to leave it in